### ISSUE TITLE
Feature request: choosing python executable

### ISSUE THREAD

**Issue by golinski** [retext-project/retext#200]
It would be very helpful, if one could choose which executable to use for compiling the documents, e.g., from a python virtualenv.

**Comment by mitya57** [comment]
Can you please be more specific? Which executable?

**Comment by taozi1314** [comment]
It is ok when I use "sudo ln -s /usr/bin/python2.7 /usr/bin/python", I
think it is the reason of python .
I'm really sorry so late reply to you .  My english is poor .

**Comment by golinski** [comment]
Taozi1314's comment is not what I meant. It's not about choosing python2 over python3 etc.

I have a bunch of files for my students, made with reStructuredText and compiled with rst2html5 (https://pypi.python.org/pypi/rst2html5). rst2html5 is not installed system-wide, together with Pygments it is installed in a virtualenv to avoid clutter. I've not known about retext, so for authoring I've used the simple combination of vim and make. I've just discovered retext and thought it would be nice to use, at least for the live preview. Unfortunately I can't because there is no option to use a specific tool for compilation.

After posting the issue I've taken a look around the application and the pymarkups module, and I can now see that this feature request would be more problematic than I thought. I thought is uses subprocesses of some kind, but now I see that for compilation it uses directly the python parsers API. So this feature would be probably more work than it's worth.
